web-app-router/tailwind.config.ts

```diff
@@ -6,6 +6,7 @@ const config: Config = {
     "./components/**/*.{js,ts,jsx,tsx,mdx}",
     "./app/**/*.{js,ts,jsx,tsx,mdx}",
     "./scenes/**/*.{js,ts,jsx,tsx,mdx}",
+    "node_modules/flowbite-react/lib/esm/**/*.js",
```

Comment:
@andy-t-wang why do we need this?

Reply:
Required to use flowbite

Reply:
https://www.flowbite-react.com/docs/getting-started/quickstart